app/components/ui/headers.tsx

```diff
@@ -0,0 +1,7 @@
+export function H1({ children }: { children: React.ReactNode }) {
```

Comment:
Looks great! Thanks for creating the file!

> Should we limit the content to be string type? Would it be weird if people wrapped a component in the H1 /H2 tag?

I like the `children` implementation that you currently have. It matches how the `h1`, `h2` HTML elements work and also allows us to nest `span`, `a` or other elements within the headings.